### PR TITLE
Aligning the options horizontally in mobile view wherever required

### LINKED ISSUE #2541: Aligning the options to preview, edit event boxes horizontally in mobile view
**Is your feature request related to a problem? Please describe.**

The icons of the boxes shown in the previewing the event are shown vertically taking a lot space in the mobile view.

**Describe the solution you'd like**

To align them horizontally

**Describe alternatives you've considered**

We can also use drop down menu for this thing

**Screenshot**
https://photos.app.goo.gl/4a4H58WeBhqxJPv98

**Additional context**

Share your views, I would like to work on it.

### PR DESCRIPTION
**Pull request by mohitm15**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
It displays the option box like `edit`,`view`,`delete`,`activate`,`deactivate` on the event overview horizontally in the mobile view.

#### Changes proposed in this pull request:

- Update files by adding conditions for mobile view. Changed files are listed in the pull request.

#### Screenshot

Screenshot 2019-04-13 22:41:48

Fixes #2541

### CODE REVIEW

**Comment by mohitm15** [comment]
@CosmicCoder96 @ritikamotwani Review this!!

File: app/templates/components/ui-table/cell/cell-simple-buttons.hbs [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
-  {{#ui-popup content=(t 'Edit') class='ui icon button' click=(action editSession record.id record.event.id) position='left center'}}
-    <i class="edit icon"></i>
-  {{/ui-popup}}
-  {{#ui-popup content=(t 'Delete') click=(action (confirm (t 'Are you sure you would like to delete this Session?') (action deleteSession record))) class='ui icon button' position='left center'}}
-    <i class="trash outline icon"></i>
-  {{/ui-popup}}
-</div>
+{{#if device.isMobile}}
+  <div class="ui horizontal compact basic buttons">
+    {{#ui-popup content=(t 'View') class='ui icon button'  click=(action viewSession record.id) position='left center'}}
```
Comment (ritikamotwani) [inline app/templates/components/ui-table/cell/cell-simple-buttons.hbs]: Don't repeat the entire code in if and else.
There's only one word (horizontal/vertical) that is getting changed. Shouldn't you be comparing just for that?

Comment (mohitm15) [reply app/templates/components/ui-table/cell/cell-simple-buttons.hbs]: Please review @ritikamotwani

**Comment by mohitm15** [comment]
@ritikamotwani @CosmicCoder96   Please review this, I had made the necessary changes.

**Comment by mohitm15** [comment]
> you just need to replace the word `vertical` not the entire list of classes.

@CosmicCoder96 Now review sir, I had made changes :smiley:

**Comment by mohitm15** [comment]
@CosmicCoder96 Please review this

**Comment by mohitm15** [comment]
I had reverted the last commit of `indentation error`. committed accidentally here

**Comment by mohitm15** [comment]
@CosmicCoder96 Please Review this

**Comment by mohitm15** [comment]
> Squash you commits @mohitm15

@CosmicCoder96  Sir I tried to rebase the HEAD but it shows merge conflicts , so I solve them manually but as soon as I again does rebase , it again shows the merge conflicts .
So kindly ignore the commits and merge this.

**Comment by kushthedude** [comment]
Squashing the commits plays important role buddy , And the merge conflicts are because your remote and local branch are not in sync , You can open a new branch in your local and force push the it in your branch `branch6` .

**Comment by mohitm15** [comment]
@CosmicCoder96 Please have a look

**Comment by CosmicCoder96** [comment]
@mohitm15 You really need to learn to squash your commits, I feel we are having this discussion more than required. Ask your peers to help you out with it.

**Comment by mohitm15** [comment]
@CosmicCoder96 sure

**Comment by kushthedude** [comment]
@mohitm15 Status?

**Comment by mohitm15** [comment]
@kushthedude working on it

**Comment by kushthedude** [comment]
How much time you need to squash the commit ? @mohitm15 its been 4 days

**Comment by mohitm15** [comment]
@pradeepgangwar review this sir

**Comment by CosmicCoder96** [comment]
As mentioned before, learn to squash your commits before making PR, and especially if reviewers explicitly requested that.

**Pull request closed without merging**